packages/tools/piw-utils-internal/src/utils/PageEditorUtils.ts

```diff
@@ -107,3 +107,20 @@ function modifyProperty(
         });
     });
 }
+
+export function moveProperty(fromIndex: number, toIndex: number, properties: Properties): void {
```

Comment:
Inst better to use the caption instead of the index?

Reply:
Either would be fine I think, but I thought that for starters we should stick to the indices. I think it makes more sense for the utility function to say "move property from index 3 to 2", rather than "move Dimensions property to Data source property". The latter is a bit more confusing imo.

The user would then be responsible for handling incorrect properties or indices, rather than this function. But we could always create an abstraction on top of this utility function if we think that in the future using the indices is too low level

Reply:
Agreed